### PR TITLE
tbi(telemetry): update init settings, bump azure applicationinsights and stablise tests

### PR DESCRIPTION
**Pull request by cianmSAP**
https://github.com/SAP/open-ux-tools/issues/1417

- Adds `watchTelemetrySettingStore` as a required option to ToolsSuiteTelemetryInitSettings. Using a watch on the store is not always desirable

- Bump `applicationinsights` to `2.9.2` to incorporate fix https://github.com/microsoft/ApplicationInsights-node.js/pull/1241 displaying `ApplicationInsights:Invalid JSON config file when using APPLICATIONINSIGHTS_CONNECTION_STRING` error 

- Increasing timeout for tests due to instabilities. Also making paths dynamic so they can be ran with tools like jest-runner

### CODE REVIEW

File: packages/telemetry/test/tools-suite-telemetry/index.test.ts [hunk 1]
```diff
@@ -30,6 +31,8 @@ jest.mock('axios', () => {
 });
 
 describe('Tools Suite Telemetry Tests', () => {
+    jest.setTimeout(10000);
```
Comment (IainSAP) [inline packages/telemetry/test/tools-suite-telemetry/index.test.ts:34]: 👍

**Review by IainSAP — APPROVED** [review]
Thanks @cianmSAP 

- Timeout inc. to avoid timeout on long running tests. 
- Nothing to be tested

**Comment by cianmSAP** [comment]
> @cianmSAP do you know a little more about the instabilities? What is taking so long?

After some investigation I can see that `getAppProperties` https://github.com/SAP/open-ux-tools/blob/0682c012850d1b14401d54957ba65c479610561c/packages/telemetry/src/tooling-telemetry/data-processor.ts#L98

can take a 3-4 seconds to complete in some cases (atleast on my machine). I think the timeout here covers this unpredictability 

@tobiasqueck

File: packages/telemetry/src/tooling-telemetry/types.ts [hunk 2]
```diff
@@ -11,12 +11,14 @@ export type TelemetryHelperProperties = {
...
  * - consumerModule: name and version of module that uses telemetry library
  * - internalFeature: if UX tooling InternalFeature is enabled
+ * - watchTelemetrySettingStore: watch changes to telemetry setting in the store and update runtime settings accordingly (recommended for extensions)
  * - resourceId: Id of cloud telemetry resource (e.g. Azure application insights resource is supported).
  *
  */
 export type ToolsSuiteTelemetryInitSettings = {
     consumerModule: ProjectInfo;
     internalFeature: boolean;
+    watchTelemetrySettingStore: boolean;
```
Comment (devinea) [inline packages/telemetry/src/tooling-telemetry/types.ts:21]: @cianmSAP This should probably updated also https://github.com/SAP/open-ux-tools/blob/5125f51449b8644270f72476f6d6ea172a31d5ec/packages/telemetry/example/index.ts#L43

Comment (cianmSAP) [reply packages/telemetry/src/tooling-telemetry/types.ts:21]: Thanks @devinea

**Review by devinea — APPROVED** [review]
Thanks for the update. 
updates are clear and covered by tests. 
changeset ✅

**Pull request merged**